src/wp-content/themes/twentyfifteen/js/functions.js

```diff
@@ -5,130 +5,162 @@
+			} );
 		} );
 	}
-	initMainNavigation( $( '.main-navigation' ) );
+	initMainNavigation( document.querySelector( '.main-navigation' ) );
 
 	// Re-initialize the main navigation when it is updated, persisting any existing submenu expanded states.
-	$( document ).on( 'customize-preview-menu-refreshed', function( e, params ) {
-		if ( 'primary' === params.wpNavMenuArgs.theme_location ) {
-			initMainNavigation( params.newContainer );
-
-			// Re-sync expanded states from oldContainer.
-			params.oldContainer.find( '.dropdown-toggle.toggle-on' ).each(function() {
-				var containerId = $( this ).parent().prop( 'id' );
-				$( params.newContainer ).find( '#' + containerId + ' > .dropdown-toggle' ).triggerHandler( 'click' );
-			});
-		}
-	});
+	// This is only relevant for the Customize preview where jQuery is expected to be loaded.
+	if ( window.jQuery ) {
+		var $ = window.jQuery;
+		$( document ).on( 'customize-preview-menu-refreshed', function( e, params ) {
+			if ( 'primary' === params.wpNavMenuArgs.theme_location ) {
+				// Extract the raw DOM element from the jQuery wrapper here.
+				initMainNavigation( params.newContainer[0] );
+
+				// Re-sync expanded states from oldContainer.
+				params.oldContainer.find( '.dropdown-toggle.toggle-on' ).each(function() {
+					var containerId = $( this ).parent().prop( 'id' );
+					$( params.newContainer ).find( '#' + containerId + ' > .dropdown-toggle' ).triggerHandler( 'click' );
+				});
+			}
+		});
+	}
 
-	secondary = $( '#secondary' );
-	button = $( '.site-branding' ).find( '.secondary-toggle' );
+	secondary = document.getElementById( 'secondary' );
+	button = document.querySelector( '.site-branding .secondary-toggle' );
 
 	// Enable menu toggle for small screens.
 	( function() {
 		var menu, widgets, social;
-		if ( ! secondary.length || ! button.length ) {
+		if ( ! secondary || ! button ) {
 			return;
 		}
 
 		// Hide button if there are no widgets and the menus are missing or empty.
-		menu    = secondary.find( '.nav-menu' );
-		widgets = secondary.find( '#widget-area' );
-		social  = secondary.find( '#social-navigation' );
-		if ( ! widgets.length && ! social.length && ( ! menu.length || ! menu.children().length ) ) {
-			button.hide();
+		menu    = secondary.querySelector( '.nav-menu' );
+		widgets = secondary.querySelector( '#widget-area' );
+		social  = secondary.querySelector( '#social-navigation' );
+		if ( ! widgets && ! social && ( ! menu || ! menu.children.length ) ) {
+			button.style.display = 'none';
 			return;
 		}
 
-		button.on( 'click.twentyfifteen', function() {
-			secondary.toggleClass( 'toggled-on' );
-			secondary.trigger( 'resize' );
-			$( this ).toggleClass( 'toggled-on' );
-			if ( $( this, secondary ).hasClass( 'toggled-on' ) ) {
-				$( this ).attr( 'aria-expanded', 'true' );
-				secondary.attr( 'aria-expanded', 'true' );
+		button.addEventListener( 'click', function() {
```

Comment:
the click handler here is slightly different than above (which targets the twentyfifteen class, will that change anything (maybe not)?

Reply:
It doesn't have to do with a CSS class, it's a [jQuery namespace](https://api.jquery.com/event.namespace/). This is however pointless to have here since it's not handled in any special way, so a regular `click` event suffices.